packages/core/bootstrap/src/lib/util.ts

```diff
@@ -225,6 +227,52 @@ const permutations = (collection: any, n: any) => {
  */
 export const permutator = (options: string[], delimiter?: string): string[] | string[][] => {
   const output: string[][] = flatMap(options, (_: any, i: any, a: any) => permutations(a, i + 1))
-  const join = (combos: string[][]) => combos.map((p) => p.join(delimiter))
+  const join = (combos: string[][]) => combos.map(p => p.join(delimiter))
   return typeof delimiter === 'string' ? join(output) : output
 }
+
+/**
+ * @description
+ * Redacts secrets from JSON object (https://github.com/winstonjs/winston/issues/1079#issuecomment-825115754)
+ *
+ * @param obj The object to redact secrets from
+ *
+ * @returns redacted object
+ */
+export const redact = (obj: any): any => {
+  const copyObj = (obj: any): any => JSON.parse(JSON.stringify(obj))
```

Comment:
We could have some data loss during the `stringify`. Values as `undefined` will be lost, or Dates will be stringified.
You can use lodash, is already used is other places, with `_.cloneDeep`